ci/manifest.fr.yml

```diff
@@ -4,4 +4,5 @@ domain: fr.cloud.gov
 # TODO make fr.cloud.gov an `owned` domain
 # https://github.com/18F/cg-landing/pull/74
 no-hostname: false
+host: www
```

Comment:
@dlapiduz Hmm, this deployed as expected, and the config and `curl` response look as expected:

```
$ cf apps
Getting apps in org cloud-gov / space landing as admin...
OK

name      requested state   instances   memory   disk   urls
landing   started           ?/1         64M      1G     landing.fr.cloud.gov, www.fr.cloud.gov
$ curl -I https://www.fr.cloud.gov
HTTP/1.1 200 OK
Accept-Ranges: bytes
Content-Length: 18408
Content-Type: text/html; charset=utf-8
Date: Fri, 10 Jun 2016 21:58:17 GMT
Etag: "575b36d6-47e8"
Last-Modified: Fri, 10 Jun 2016 21:53:26 GMT
Server: nginx/1.8.0
Strict-Transport-Security: max-age=31536000; includeSubDomains; preload
Vary: Accept-Encoding
X-Content-Type-Options: nosniff
X-Frame-Options: DENY
X-Vcap-Request-Id: bafc764b-6e26-4394-6a85-44456d13cf9c
X-XSS-Protection: 1; mode=block
Connection: keep-alive
```

For some reason, however, visiting in a browser redirects to https://fr.cloud.gov. Tried in a couple browsers and saw the same behavior. Is this an HSTS thing or something?